page/basics/loops/index.markdown

```diff
@@ -2,23 +2,30 @@
 
 #### foreach
 
-Traditional `for` loops exist but you almost never need them in Perl.
-`foreach` is usually simpler.
+C-like `for` loops exist but you almost never need them in Perl.
+`foreach` is usually simpler (but can be spelled as either `for`
+or `foreach`).
 
 Iterating over an array
 
-    my @array = (1, 2, 3, 4);
+    my @array = ('a', 'b', 'c', 'd');
```

Comment:
Just curious, why you changed this.  Is it more intuitive to you for some reason?

Reply:
Not more intuitive, but an example of other stuff in the array.   By saying `for my $i (@array) { ... }` there may be some implication that may cause confusion.  It `$i` an index into the array?  Is `$i` an integer?  I just wanted to show that there could be other things to allay some of that.

Reply:
Thanks that makes sense